modules/ROOT/pages/cloudhub-faq.adoc

```diff
@@ -161,28 +193,10 @@ Because CloudHub deploys on Amazon EC2, IP addresses are chosen from the Amazon
 
 Customers with an Enterprise account can choose any global region in which to deploy their app. The actual worker is always running in the UTC timezone.
 
-== How do I get support?
-
-Browse and search our online http://forums.mulesoft.com[forum] archives to find answers. Or post a question and start a new thread.
-
-The CloudHub team is committed to providing the best customer experience possible. Contact the CloudHub team in the https://help.mulesoft.com[MuleSoft Help Center].
-
-If you have a paid plan, you also have access to the support portal. To file a case through the support portal:
-. Sign into the Anypoint Platform, then click *Support* on the top nav bar.
-. Click the *Cases* tab.
-+
-You can browse open cases or click *Create New Case* to open a new case.
-. When you create a new case, fill in the required fields (marked in red).
-+
-Hover over icons for field-specific guidelines or instructions.
-. When finished, click *Submit*.
-+
-The portal generates a unique case number and refer you to possibly relevant areas of the knowledge base that you can browse while you await a response to your case.
-
 
 == Proactive Monitoring
 
-The CloudHub platform is monitored 24 hours a day, 7 days a week, by automated systems. In the event of any issue affecting the health and operation of the CloudHub infrastructure, our dedicated operations team is notified and responds immediately to diagnose and correct it. This 24x7 monitoring covers the entire CloudHub platform, benefiting all CloudHub users, free and paying. The status of CloudHub services and upcoming maintenance can be found at http://status.mulesoft.com[`status.mulesoft.com`].
+The CloudHub platform is monitored 24 hours a day, 7 days a week, by automated systems. In the event of any issue affecting the health and operation of the CloudHub infrastructure, our dedicated operations team is notified and responds immediately to diagnose and correct it. This 24x7 monitoring covers the entire CloudHub platform, benefiting all CloudHub users, free and paying. The status of CloudHub services and upcoming maintenance can be found at http://status.mulesoft.com[`status.mulesoft.com`] and http://eu1-status.mulesoft.com[`eu1-status.mulesoft.com`].
```

Comment:
I would normally suggest making this more concise and less sales-oriented, but since your Dev agonized over it...meh.... Here are a couple you might slip through:
```suggestion
The CloudHub platform is monitored 24 hours a day, 7 days a week, by automated systems. Should any issue affect the health of the CloudHub infrastructure, the operations team is notified and responds immediately to diagnose and correct it. This 24x7 monitoring covers the entire CloudHub platform, benefiting all CloudHub users. You can view the current status of CloudHub services and upcoming maintenance at http://status.mulesoft.com[`status.mulesoft.com`] and http://eu1-status.mulesoft.com[`eu1-status.mulesoft.com`].
```